tests/test_other.py

```diff
@@ -7339,14 +7339,6 @@ def test_binaryen_invalid_method(self):
     proc.communicate()
     assert proc.returncode != 0
 
-  def test_binaryen_default_method(self):
```

Comment:
What do you think about putting the removed loggings behind RUNTIME_LOGGING, and using that in the test? Then we don't lose any coverage, and we've still gotten rid of all the annoying logging.

Oh, also please add yourself to AUTHORS if you're not there already.